src/python/WMCore/REST/Format.py

```diff
@@ -35,6 +34,9 @@ def is_iterable(obj):
         return True
 
 class RESTFormat:
+    def __init__(self):
```

Comment:
Like we said in our emails, I think inheriting from object will fix this with no other changes